package.json

```diff
@@ -16,7 +16,9 @@
     "start-prof": "node --prof --logfile=${LEDGER_V8_LOGFILE:-v8.log} src/app.js",
     "lint": "eslint .",
     "test": "istanbul test -- _mocha",
-    "test-oracle": "LEDGER_UNIT_DB_URI='oracle://system:oracle@192.168.99.100:49161/' DYLD_LIBRARY_PATH=/opt/oracle/instantclient LD_LIBRARY_PATH=/opt/oracle/instantclient node node_modules/.bin/istanbul test -- _mocha",
+    "sqlplus": "LD_LIBRARY_PATH=/opt/oracle/instantclient DYLD_LIBRARY_PATH=/opt/oracle/instantclient /opt/oracle/instantclient/sqlplus system/oracle@localhost:49161/xe",
+    "setup-oracle": "echo exit | LD_LIBRARY_PATH=/opt/oracle/instantclient DYLD_LIBRARY_PATH=/opt/oracle/instantclient /opt/oracle/instantclient/sqlplus system/oracle@localhost:49161/xe @src/sql/strong-oracle/create.sql",
+    "test-oracle": "LEDGER_UNIT_DB_URI='oracle://system:oracle@localhost:49161/xe' DYLD_LIBRARY_PATH=/opt/oracle/instantclient LD_LIBRARY_PATH=/opt/oracle/instantclient node node_modules/.bin/istanbul test -- _mocha",
```

Comment:
You mean _to_ `localhost`? It's because we are using new oracle setup instructions.